packages/core/src/components/widgets/CheckboxWidget.tsx

```diff
@@ -53,17 +52,4 @@ CheckboxWidget.defaultProps = {
   autofocus: false,
 };
```

Comment:
You can delete the `defaultProps` since you've applied the `autofocus=true` above